.github/workflows/ci.yaml

```
@@ -39,6 +39,33 @@ jobs:
           # Setting LD_LIBRARY_PATH is a workaround for https://github.com/TheBevyFlock/bevy_new_2d/pull/318#issuecomment-2585935350
           LD_LIBRARY_PATH="$(rustc --print target-libdir)" cargo test --locked --workspace --all-features --doc
 
+  # Check that the game builds for Wasm.
+  build-wasm:
+    name: Build for Wasm
```

```suggestion
  # Check that the game builds for web.
  build-web:
    name: Build for web
```
Parallel with `bevy build web`